app/views/products/_details_text.haml

```diff
@@ -1,8 +1,8 @@
 - if not @product.origin.empty?
   %h5.subhead Origin
-  %p.origintext
+  %p.subheadtext.origintext
     = @product.origin
 - if not @product.cultural_history.empty?
   %h5.subhead Cultural History
-  %p.subheadtext
-    = @product.cultural_history
\ No newline at end of file
+  %p.subheadtext.culturaltxt
```

Comment:
Slight inconsistency in naming of "culturaltxt" vs "subheadtext". Should probably be "culturaltext" instead.